.github/workflows/prove.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Perl tests
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  fullstack:
+    runs-on: ubuntu-latest
```

Comment:
I think you should add a name in this level. See how it's currently named in the rest results in the main PR page, something about JavaScript, not that helpful

Reply:
Not sure what you're asking. `fullstack` shows up as the name of this job:

![image](https://user-images.githubusercontent.com/1204189/142827278-1d5dfc60-edac-464c-8d61-1a06598e01e0.png)

Reply:
I mean here

![github_pr_details_name_Screenshot_20211122_101304](https://user-images.githubusercontent.com/1693432/142834330-b7f0d487-5284-4697-b870-b08011630a41.png)

where it says "javascript / Node 16.x and ubuntu-latest"

Reply:
That doesn't have anything to do with this pull request, though.

Reply:
oh, sorry. I misread. The new test says "Perl tests / fullstack". Ok, that's fine. Disregard this point